.teamcity/common/performance-test-extensions.kt

```diff
@@ -50,3 +50,9 @@ fun performanceTestCommandLine(task: String, baselines: String, extraParameters:
 fun distributedPerformanceTestParameters(workerId: String = "Gradle_Check_IndividualPerformanceScenarioWorkersLinux") = listOf(
         "-Porg.gradle.performance.buildTypeId=$workerId -Porg.gradle.performance.workerTestTaskName=fullPerformanceTest -Porg.gradle.performance.coordinatorBuildId=%teamcity.build.id% -PgithubToken=%github.ci.oauth.token%"
 )
+
+val individualPerformanceTestArtifactRules = """
+        subprojects/*/build/test-results-*.zip => results
+        subprojects/*/build/tmp/**/log.txt => failure-logs
+        subprojects/*/build/tmp/**/profile.log => failure-logs
```

Comment:
Are we by any chance leaving behind `profile-out-7` etc. directories from previous runs that will be collected here? (Probably not, right?)

Reply:
No, we set the output directory explicitly: https://github.com/gradle/gradle/blob/2136f7d957f40e8546f64bb4940522c57d4d8b77/subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/GradleProfilerBuildExperimentRunner.java#L144-L146

The `tmp` directory should be cleaned up by `clean` as well, which we run on CI.